IndomitableIbises/Bangazon-Sprint-2
# Ls style

## Link to Ticket
references # [Link](https://github.com/IndomitableIbises/Bangazon-Sprint-2/issues/33)

## Description of Proposed Changes
- styles the index page and navigation
- changed the text on the home page & centered and justified it
- also fixed the gitignore to properly ignore the django_data.sh

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout lsStyle
```
1. makemigrations and migrate
1. run the server and check the various pages
1. the navigation should be centered and inline on all pages
1. text on the home page should be centered and justified


## Impacted Areas in Application

List general components of the application that this PR will affect:
- nav templates
- index template
- new file app/bang/static/bang/style.css

## Mentions @username

Tag users that need to review this code

## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
